### ISSUE TITLE
Ram loss porting through cave 

### ISSUE THREAD

**Issue by Candeyy** [WarEmu/WarBugs#19433]
In transporting a ram from Marsh if Madness to Barak Varr, intermittently, the ram won’t come through the cave to BV side and is lost. I was in a warband where they had to purchase a new one and bring it BV the long route (playing in Order side).

**Expected behavior and actual behavior:**

The above shouldn’t occur, ram should come through cave with the owner.

**Steps to reproduce the problem:**

1. Purchase a ram from Marsh of Madness keep.
2. Attempt to bring it through the cave.
3. Ram doesn’t always come through cave on Barak Varr side.
4. NOTE: This bug is intermittent.

**Testing Screenshots/Videos/Evidences (always needed):**

Sorry, no screenshot.

**Comment by CrimsonFenris** [comment]
Yes, I noticed (coincidence or not) ram to disappear when it's moving. Try to stay stationary near tunnel, wait for ram to stay still, than click the tunnel.

Also this bug will often debuff your movement speed permanently - the game still counting you got the ram attached (relogging fixes the issue).
